cursorless-talon-dev/src/default_vocabulary.py

```diff
@@ -96,14 +102,17 @@
     "pound": "£",
 }
 
-any_alphanumeric_keys = {
+ctx.lists["user.cursorless_default_any_alphanumeric_key"] = {
     **{w: chr(ord("a") + i) for i, w in enumerate(initial_default_alphabet)},
     **{digits[i]: str(i) for i in range(10)},
     **punctuation_words,
     **symbol_key_words,
 }
 
 
-@ctx.capture("user.any_alphanumeric_key", rule="|".join(any_alphanumeric_keys.keys()))
+@ctx.capture(
+    "user.any_alphanumeric_key",
+    rule="{user.cursorless_default_any_alphanumeric_key}",
```

Comment:
Our capture parser can only deal with lists and not literals
https://github.com/cursorless-dev/cursorless/blob/e9160e95cca55ab8b86ab290acf5f35254cd062f/cursorless-talon/src/get_grapheme_spoken_form_entries.py#L37-L68